### ISSUE TITLE
Refactor Inspector tabs for accessibility

### ISSUE THREAD

**Issue by swissspidy** [GoogleForCreators/web-stories-wp#933]
## Feature Description

At some point the library (left-hand side) was refactored for better keyboard accessibility.

The same should be done for the inspector (right-hand side) as well.

Lots of potential to abstract this to make it more DRY.

Tests needed to verify intended behavior.

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._
